### ISSUE TITLE
OTG selecting a USB results in Amaze completely losing its Styling. 

### ISSUE THREAD

**Issue by Krammig** [TeamAmaze/AmazeFileManager#1279]
<img src="https://user-images.githubusercontent.com/11222718/41326590-f82f2610-6f02-11e8-9ecb-8e3f47620fd2.png" height=500px /><img src="https://user-images.githubusercontent.com/11222718/41326601-03bd67ee-6f03-11e8-8071-5bdeaf9f3be9.png" height=500px />

The phone Sony Xperia Z5 running 7.1.1

Local Storage
SDCard
OTG  (USB 32Gb plug in) 

Both the Local Storage and the SDCard operate fine in terms of navigation and the correct Amaze style, colours ans layout. 

However selecting OTG from the slide out menu, displays what looks like the default Android explorer. Just black and white, squares for folder and files.
A message does flash up for a moment saying "Select the Root of OTG device"  No idea what it is trying to say here or do. 

Bottom line, Amaze does not operate correctly on OTG. But surely this would have been discovered a long while ago ?  
Am I doing something wrong ?

**Comment by EmmanuelMess** [comment]
We should explain better, thanks for bringing this up.

You should open the app drawer (three bars button above), select the USB device and press "select" button (lower right corner).

**Comment by EmmanuelMess** [comment]
This issue is closed, but the path not changing is a bug. You can open another issue with that, or I do it later.
